gravitee-apim-portal-webui-next/src/services/api.service.ts

```diff
@@ -30,10 +30,11 @@ export class ApiService {
     private configService: ConfigService,
   ) {}
 
-  list(page = 1, size = 9): Observable<ApisResponse> {
+  list(page = 1, category: string, size = 9): Observable<ApisResponse> {
     return this.http.get<ApisResponse>(`${this.configService.baseURL}/apis`, {
       params: {
         page,
+        category: category !== 'all' && category !== undefined ? category : [],
```

Comment:
A user should only be able to specify one category. So instead of returning `[]`, you can return an empty string.